classes/CustomerThread.php

```diff
@@ -25,6 +25,11 @@
  */
 class CustomerThreadCore extends ObjectModel
 {
+    const openStatus = 'open';
+    const closedStatus = 'closed';
+    const pending1Status = 'pending1';
+    const pending2Status = 'pending2';
```

Comment:
These statuses are introduced here https://github.com/PrestaShop/PrestaShop/pull/14288/files#diff-7a6118dbd41aee4add4f5f04d7bd3f35R34 

I prefer to use `CustomerThreadStatus` VO instead of this, @matks wdyt?

Reply:
I think you're totally right @sarjon , I had no knowledge of this, I'll update the PR and use these constants.